sites/kit.svelte.dev/src/routes/docs/index.svelte

```diff
@@ -22,4 +22,5 @@
 	<meta name="Description" content="Complete documentation for SvelteKit">
 </svelte:head>
 
+<h1 class="visually-hidden">Docs</h1>
```

Comment:
Maybe "Documentation"? We probably just went with "Docs" in the nav to save space

Reply:
Updated in latest commit.